src/System.ComponentModel.Annotations/tests/MinLengthAttributeTests.cs

```diff
@@ -49,6 +56,34 @@ public static void Ctor(int length)
             Assert.Equal(length, new MinLengthAttribute(length).Length);
         }
 
+        [Theory]
+        [MemberData(nameof(ValidValues_ICollection))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "MinLengthAttribute in the .NET Framework doesn't support ICollection.Count. See https://github.com/dotnet/corefx/pull/2650")]
```

Comment:
Could you just please add the issue number to the SkipOnTargetFrameworkAttribute so that is easy to track when we look for the tests being skipped?